fastlane/Fastfile

```diff
@@ -60,6 +60,40 @@ lane :version_bump do |options|
   )
 end
 
+
+desc "Bump version in all necessary files"
+lane :bump do |options|
+  version = options[:version]
+  puts "Version from parameter: " + version
+
+  version_bump_podspec(path: ENV['podspec_file'], version_number: version)
+end
+
+desc "Release a new snapshot"
+lane :release_snapshot do |options|
+  version = options[:version].to_s
+  UI.user_error!("You need to provide the new version number like: make release_snapshot version='X.X.X'") if version.strip.empty?
```

Comment:
Hey @caiobzen and @paulogama14, maybe we can ensure the version of the release contains the word snapshot.

What do you think?

Reply:
Great idea. 

=]